Dockerfile

```diff
@@ -13,4 +13,3 @@ ARG BASE_PACKAGE_NAME
 COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=0 /go/src/$BASE_PACKAGE_NAME/bin/broker /app/broker
 CMD ["/app/broker"]
-EXPOSE 8080
```

Comment:
May wonder why this line is going away...

This PR adds the flexibility for the broker to run on _any_ port, dependent on the value of the `API_SERVER_PORT` env var. (The chart will pick 8080 or 8443 depending on if TLS is being used or not-- and really those are just for vanity.) But since the software on the Docker image can use any port, there's no point in keeping the directive `EXPOSE 8080`, because according to Docker's own documentation, that directive doesn't have any real effect at runtime and only serves to document expectations between image authors and people who run containers.

Reply:
👍